pkg/cedar/core/client.go

```diff
@@ -119,6 +159,29 @@ func (c *Client) GetSystemSummary(ctx context.Context) ([]*models.CedarSystem, e
 	return retVal, nil
 }
 
+// populateSystemSummaryCache is a method used internally by the CEDAR Core client
+// to populate the in-memory cache with the results of a call to the /system/summary endpoint
+//
+// It does not return anything from the cache, nor does it return anything at all (unless an error occurs)
+func (c *Client) populateSystemSummaryCache(ctx context.Context) error {
+	appcontext.ZLogger(ctx).Info("Refreshing CEDAR Core cache")
+	if !c.cedarCoreEnabled(ctx) {
+		appcontext.ZLogger(ctx).Info("CEDAR Core is disabled")
+		return nil
+	}
+
+	// Get data from API - don't use cache to populate cache!
+	systemSummary, err := c.GetSystemSummary(ctx, false)
+	if err != nil {
+		return err
+	}
+
+	// Set in cache
+	c.cache.Set(allSystemsCacheKey, systemSummary, cache.DefaultExpiration)
```

Comment:
I think you can use `cache.NoExpiration` if we aren't using the default expiration

Reply:
I found an even better solution that I had no idea existed! Will commit here in a sec, but there's a `cache.SetDefault` which implicitly passes `cache.DefaultExpiration` under the hood :)

Reply:
Doesn't `cache.New()` set the default expiration anyways? Seems like we'd want to have the expiration time set in one place, rather than having the "no expiration" policy be set in both the `cache.New()` call and the `cache.Set()` call.

Reply:
Yup - I think that's what `cache.SetDefault()` does - basically is the same as `cache.Set()`, but always passes `cache.DefaultExpiration` (like this code currently does)

Reply:
82d22eb42b6d184e343614b4cf640af143caf394 addresses this pretty well, IMO!